en-US/firefox/whatsnew_67.0.5.lang

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Need to include testing info. Reference this for examples: https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/new/trailhead.lang

Reply:
I can include what the demo link will be when it gets pushed up to demo on 5/21. However, this wont be exposed to the l10n community. fr and de strings will be provided once this is in Pontoon.